pybindsrc/PyBind.cpp

```diff
@@ -0,0 +1,97 @@
+#include "ers/AnyIssue.hpp"
```

Comment:
I suggest to use `module.cpp` for the main file

Reply:
Updated in branch